### ISSUE TITLE
[Gssoc'21] Contact page not accessible for all

### ISSUE THREAD

**Issue by arpit456jain** [vikasdo/Book-Recommendation-Analysis#70]
### **Describe the bug:**
The Contact page is not accessible for all .
It can be access only after login which is not good.
I want to make it accessible for all no matter they login or not.

### **To Reproduce:**
4. See an error

### **Additional context:**

<hr/>

**Note:**
* If you want to work on an issue, you should check if it has already been assigned to anyone.

**Comment by arpit456jain** [comment]
@Akshay6890 and @vikasdo sir this issue is very crucial , complicated and time taking.

Actually  in this project the template inheritance  is used in a very very wrong way .
I don't know I am able to explain it or.
But I will try it at my best and make a video for that .

**Comment by arpit456jain** [comment]
https://youtu.be/pXfeo2jRico 
the video size was very large for uploading it here so upload it on you tube kindly watch it.

**Comment by Akshay6890** [comment]
> https://youtu.be/pXfeo2jRico
> the video size was very large for uploading it here so upload it on you tube kindly watch it.

in the header.html, include the contact link under the unauthenticated and non-authenticated section.  Hope this is clear.

**Comment by arpit456jain** [comment]
@Akshay6890 I understand what you want I already done that.
<img width="960" alt="p4" src="https://user-images.githubusercontent.com/55352601/113500802-aabdab80-953e-11eb-8f42-90093ada3f28.png">

but try to understand the problem in contact page.
Let me try again to explain
As you know contact page extends the header.html
<img width="960" alt="22" src="https://user-images.githubusercontent.com/55352601/113500840-e9536600-953e-11eb-86a9-81bf423297b0.png">
so this profile section is also inherited 
now when user is not logged in **current_user.is_authenticated ** is false so it does't create any problem
but when the user login this condition is true and it want the extra parameters  

but those extra parameters can only be given when user is looged in.

I hope you understand .
My solution -  To make a separate and independent page for contact no inheritance

**Comment by arpit456jain** [comment]
@vikasdo @Akshay6890  kindly review my PR
